src/core/ThemeProvider/ThemeProvider.test.tsx

```diff
@@ -25,20 +26,24 @@ afterEach(() => {
 });
 
 it('should respect os theme (light)', () => {
+  jest.spyOn(UseMediaQuery, 'useMediaQuery').mockReturnValueOnce(false);
```

Comment:
Wouldnt make sense to have mock beforeEach with false value to all tests? And have reference to it to change it to `true` where you need?

Reply:
ok done